content/blog/tinacms-ui-whats-next.md

```diff
@@ -16,7 +15,7 @@ It is not an application. It is not intended to be an all-in-one solution. It is
 
 In order to create an amazing inline editing experience for [tinacms.org](http://tinacms.org/) we decided to create an experimental UI. We hid the Sidebar and replaced it with a persistent Toolbar at the top of the screen. This new UI was an instant hit with our team! The Toolbar gives us easier access to common actions without having to open and close the Sidebar. We still love the Sidebar, it's easier to setup and is great when you don't want or need inline editing, but it's not always the best experience.
 
-Unfortunately you'll have to wait a little bit to get your hands on the new Toolbar. The creation of the Toolbar was incredibly exciting project for us. We were able to totally re-imagine the user experience of TinaCMS _without having to change any code in the main repository_. This allowed us to quickly experiment with the interface without considering how it could fit into the overall project. Because of this, it only took a few days to get it production ready. This project validated many of our code design decisions by showing that we could experiment and innovate on top of TinaCMS without having to tear it apart.
+Unfortunately you'll have to wait a little bit to get your hands on the new Toolbar. The creation of the Toolbar was an incredibly exciting project for us. We were able to totally re-imagine the user experience of TinaCMS _without having to change any code in the main repository_. This allowed us to quickly experiment with the interface without considering how it could fit into the overall project. Because of this, it only took a few days to get it production ready. This project validated many of our code design decisions by showing that we could experiment and innovate on top of TinaCMS without having to tear it apart.
```

Comment:
the fix was here changing from:
"the Toolbar was incredibly exciting project"
to:
"the Toolbar was **an** incredibly exciting project"

Everything else should just be formatting changes from Tina